.github/workflows/make_wheel_Linux_x86.sh

```diff
@@ -11,7 +11,13 @@ else
   export TF_NAME='tensorflow'
 fi
 
-if [ $TF_VERSION == "2.6.3" ]  || [ $TF_VERSION == "2.8.3" ] ; then
+# if tensorflow version == 2.12.0 or == 2.13.0
+if [[ "$TF_VERSION" =~ ^2\.(12|13)\.0$ ]] ; then
+  export BUILD_IMAGE="tfra/nosla-cuda11.8.1-cudnn8-ubuntu20.04-manylinux2014-python$PY_VERSION"
```

Comment:
It seems we didn't build this docker image, Do you mean here is the placeholder for 2.12.x in the future?

Reply:
Yes, but we can remove it in this MR.

Reply:
It's Okey to keep it.

Reply:
Removed condition checks for TensorFlow 2.12.0 +